apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart

```diff
@@ -392,7 +392,7 @@ class MemberCard extends StatelessWidget {
                     height: 10,
                   ),
                   (isNotEligible ||
-                          isBeneficiaryRefused ||
+                          false ||
```

Comment:
@amitverma-egov This can be removed altogether

Reply:
@vishal-egov sure